.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ 3.6, 3.7, 3.8, 3.9, ]  # latest release minus three
+        python-version: [ 3.7, 3.8, 3.9, 3.10.2]  # latest release minus three
```

Comment:
If you convert these to strings then '3.10' should ensure we test the latest point release.

`python-version: [ '3.7', '3.8', '3.9', '3.10']`

And drop the comment maybe?